docs/general/ad-filtering/create-own-filters.md

```diff
@@ -4338,18 +4378,20 @@ There are 2 ways to specify domain restrictions for non-basic rules:
 But rules with mixed style domains restriction are considered invalid. So, for example, the rule
 `[$domain=example.org]example.com##.textad` will be ignored.
 
-### non-basic `$domain` modifier limitations {#non-basic-domain-modifier-limitations}
+### Non-basic `$domain` modifier limitations {#non-basic-domain-modifier-limitations}
 
 :::caution Limitations
 
-<!-- FIXME: describe $domain modifier limitations in mv3 -->
+Since the non-basic `$domain` works the same as the basic one,
+it has the same [limitations](#domain-modifier-limitations).
 
 :::
 
 :::info Compatibility
 
 Such rules with `$domain` modifier are supported by AdGuard for Windows, AdGuard for Mac, AdGuard for Android,
-and AdGuard Browser Extension for Chrome, Firefox, Edge.
+AdGuard Browser Extension for Chrome, Firefox, Edge,
+and AdGuard Chrome for MV3.
```

Comment:
AdGuard Chrome for MV3 is inconsistent with other product names
is it final?

Reply:
It is not final, I will change it when the PMs have a decision on a name